src/fides/api/models/datasetconfig.py

```diff
@@ -339,6 +343,7 @@ def convert_dataset_to_graph(
             data_categories=(
                 set(collection.data_categories) if collection.data_categories else set()
             ),
+            partitioning=collection_partitioning,
```

Comment:
This reminds me, if we end up _removing_ support for the where clauses since it's experimental, we'll have to do it in a way that still allows DSR's in process to run:

tDSR 3.0 (which are customers should all be running by now) stores a representation of this collection on the `requesttask` backing the node in the database, so we don't have to rebuild the graph.  If details change about the partitioning, there could still be `requesttasks` that haven't run in the database with the old partitioning configuration which would fail when they are queued by the worker and the worker goes to rehydrate the collection off of the request task.

Reply:
thanks for calling that out, great edge case to consider. i don't think there's a whole lot to do right now to anticipate that future state, beyond some code comments to warn the future dev that they'll need to handle that backward compatible case, right? if you have other ideas, would love to hear them to make this as seamless as possible a transition in the (hopefully near) future!

Reply:
just a callout would be fine!